Typing for SysLogHandler has several errors

Using the `logging.handler.SysLogHandler` directly, I get several errors:

```
pysoa/common/logging.py:287: error: Argument 3 to "__init__" of "SysLogHandler" has incompatible type "Optional[int]"; expected "int"
pysoa/common/logging.py:293: error: "SyslogHandler" has no attribute "unixsocket"
pysoa/common/logging.py:293: error: "SyslogHandler" has no attribute "socktype"
pysoa/common/logging.py:332: error: "SyslogHandler" has no attribute "ident"
pysoa/common/logging.py:337: error: "SyslogHandler" has no attribute "facility"
```

All of these are incorrect. According to the source code:

- The `socktype` argument (argument 3) is defined as `socktype=None`
- The constructor sets `socktype = socket.SOCK_DGRAM` if `socktype is None`
- The constructor sets `self.unixsocket`, `self.socktype`, and `self.facility`
- `ident` is a class attribute defined on the class as follows:
  ```
  ident = ''          # prepended to all messages
  ```

--- a/stdlib/2and3/logging/handlers.pyi
+++ b/stdlib/2and3/logging/handlers.pyi
@@ -4,7 +4,7 @@ import sys
 from _typeshed import StrPath
 from logging import FileHandler, Handler, LogRecord
 from socket import SocketKind, SocketType
-from typing import Any, Callable, Dict, List, Optional, Tuple, Union
+from typing import Any, Callable, ClassVar, Dict, List, Optional, Tuple, Union
 
 if sys.version_info >= (3, 7):
     from queue import Queue, SimpleQueue
@@ -160,9 +160,9 @@ class SysLogHandler(Handler):
     if sys.version_info >= (3,):
         ident: str  # undocumented
     facility: int  # undocumented
-    priority_names: Dict[str, int]  # undocumented
-    facility_names: Dict[str, int]  # undocumented
-    priority_map: Dict[str, str]  # undocumented
+    priority_names: ClassVar[Dict[str, int]]  # undocumented
+    facility_names: ClassVar[Dict[str, int]]  # undocumented
+    priority_map: ClassVar[Dict[str, str]]  # undocumented
     def __init__(
         self, address: Union[Tuple[str, int], str] = ..., facility: int = ..., socktype: Optional[SocketKind] = ...
     ) -> None: ...

--- a/stdlib/2and3/logging/handlers.pyi
+++ b/stdlib/2and3/logging/handlers.pyi
@@ -3,7 +3,7 @@ import ssl
 import sys
 from _typeshed import StrPath
 from logging import FileHandler, Handler, LogRecord
-from socket import SocketType
+from socket import SocketKind, SocketType
 from typing import Any, Callable, Dict, List, Optional, Tuple, Union
 
 if sys.version_info >= (3, 7):
@@ -12,9 +12,6 @@ elif sys.version_info >= (3,):
     from queue import Queue
 else:
     from Queue import Queue
-
-# TODO update socket stubs to add SocketKind
-_SocketKind = int
 
 DEFAULT_TCP_LOGGING_PORT: int
 DEFAULT_UDP_LOGGING_PORT: int
@@ -158,8 +155,16 @@ class SysLogHandler(Handler):
     LOG_LOCAL5: int
     LOG_LOCAL6: int
     LOG_LOCAL7: int
+    unixsocket: bool  # undocumented
+    socktype: SocketKind  # undocumented
+    if sys.version_info >= (3,):
+        ident: str  # undocumented
+    facility: int  # undocumented
+    priority_names: Dict[str, int]  # undocumented
+    facility_names: Dict[str, int]  # undocumented
+    priority_map: Dict[str, str]  # undocumented
     def __init__(
-        self, address: Union[Tuple[str, int], str] = ..., facility: int = ..., socktype: Optional[_SocketKind] = ...
+        self, address: Union[Tuple[str, int], str] = ..., facility: int = ..., socktype: Optional[SocketKind] = ...
     ) -> None: ...
     def encodePriority(self, facility: Union[int, str], priority: Union[int, str]) -> int: ...
     def mapPriority(self, levelName: str) -> str: ...

